src/main/java/io/roastedroot/proxywasm/v1/ProxyWasm.java

```diff
@@ -302,6 +304,15 @@ public ProxyWasm build(WasmModule module) throws StartException {
 
             imports.addMemory(Objects.requireNonNullElseGet(memory, this::defaultImportMemory));
             imports.addFunction(toHostFunctions());
+            imports.addFunction(
+                    new HostFunction(
+                            "env",
+                            "emscripten_notify_memory_growth",
```

Comment:
I think there are ways to make EMScripten not emit this import, but this was the quickest hack to get everything working.